spec/serializer_spec.rb

```diff
@@ -251,6 +269,23 @@ class FooChildSerializer < FooSerializer
         end
       end
     end
+
+    describe 'collection of Struct' do
```

Comment:
this example group should be inside `describe 'collection'` describe block (line 233)
and it should be named 'collection of structs'